### PR TITLE
Updates for QuicTrace

### PR DESCRIPTION
**Pull request by nibanks**
## Description

Lots of improvements for quictrace

## Testing

N/A

## Documentation

N/A

### CODE REVIEW

File: src/plugins/trace/exe/Program.cs [hunk 1]
```diff
@@ -229,11 +232,41 @@ public void UpdateToState(RequestState state, ulong time, bool allowPrevious = f
...
+                LastStateChangeTime = time;
+            }
+
             //
             // Returns the sum of all the calculated times for individual layers.
             //
             public ulong TotalTime { get { return Times.Aggregate((temp, x) => temp + x); } }
 
+            //
+            // An estimate of the network cost from (Client.FirstSend-Client.FirstRecv)-(Server.FirstRecv-Server.FirstSend)
```
Comment (csujedihy) [inline src/plugins/trace/exe/Program.cs:254]: (Client.FirstSend-Client.FirstRecv) => (Client.FirstRecv - Client.FirstSend)?

Comment (csujedihy) [inline src/plugins/trace/exe/Program.cs:254]: (Client.FirstSend-Client.FirstRecv) => (Client.FirstRecv - Client.FirstSend)?

Comment (nibanks) [reply src/plugins/trace/exe/Program.cs:254]: Nope. It's correct as written.

Comment (nibanks) [reply src/plugins/trace/exe/Program.cs:254]: Oh, actually both are reversed.

File: src/plugins/trace/exe/Program.cs [hunk 2]
```diff
@@ -921,7 +921,7 @@ static void Main(string[] args)
                 {
                     VerboseMode = true;
                 }
-                else
+                elseIdl
```
Comment (csujedihy) [inline src/plugins/trace/exe/Program.cs:924]: elseIdl?

Comment (nibanks) [reply src/plugins/trace/exe/Program.cs:924]: weird. how'd that get there?

**Review by csujedihy — APPROVED** [review]

**Pull request merged**